components/server/leeway.Dockerfile

```diff
@@ -12,7 +12,8 @@ WORKDIR /app
 RUN /installer/install.sh
 
 FROM golang:1.17.2 as oci-tool-builder
-RUN go install github.com/csweichel/oci-tool@latest
+# cmp: https://github.com/csweichel/oci-tool/releases
+RUN go install github.com/csweichel/oci-tool@6f5c007
```

Comment:
I just pushed a semver compatible tag: v0.1.1 ... now this works:
```
go install github.com/csweichel/oci-tool@v0.1.1
```

Reply:
Done.